### ISSUE TITLE
Railway deploy fails 

### ISSUE THREAD

**Issue by d3ward** [umami-software/umami#1704]
Hello , need help to understand how to fix that error that is blocking me to deploy umami on Railway
I used the recommended deploy button , but also tried with manual deploy and issue persist

```
#16 2.074 Database type detected: postgresql
#16 2.094 Copied /app/db/postgresql to /app/prisma
 
#16 2.420 $ prisma generate
 
#16 5.180 Prisma schema loaded from prisma/schema.prisma
 
#16 5.681 Error: Unable to establish a connection to query-engine-node-api library. It seems there is a problem with your OpenSSL installation!
#16 5.681 Details: Unable to require(`/app/node_modules/prisma/libquery_engine-linux-musl.so.node`)
#16 5.681  Error loading shared library libssl.so.1.1: No such file or directory (needed by /app/node_modules/prisma/libquery_engine-linux-musl.so.node)
#16 5.681 [Context: getConfig]
#16 5.681
#16 5.681 Prisma CLI Version : 4.5.0
#16 5.710 error Command failed with exit code 1.
#16 5.710 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
 
#16 5.723 ERROR: "build-db-client" exited with 1.
#16 5.744 error Command failed with exit code 1.
 
#16 5.744 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
#16 5.758 ERROR: "build-db" exited with 1.
#16 5.789 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
#16 5.791 error Command failed with exit code 1.
 
#16 ERROR: executor failed running [/bin/sh -c yarn build-docker]: exit code: 1
-----
> [builder 5/5] RUN yarn build-docker:
-----
executor failed running [/bin/sh -c yarn build-docker]: exit code: 1
```

**Comment by mikecao** [comment]
It says right there, `It seems there is a problem with your OpenSSL installation!`. You'll have to contact Railway about it.

**Comment by sparky005** [comment]
> It says right there, It seems there is a problem with your OpenSSL installation!. You'll have to contact Railway about it.

Ok that's fair, but the docs should not recommend that we use Railway then. The docs prominently feature using Railway to deploy, but that method is currently broken. It's not a great first impression of the product for new users.

**Comment by sparky005** [comment]
@d3ward @mikecao this is the actual issue and fix: https://github.com/prisma/prisma/issues/16553#issuecomment-1353302617

I added that `apk install` line to both the builder and the runner in the Dockerfile, though I'm not sure it's actually needed in the runner.

**Comment by aidfulAI** [comment]
The `RUN apk add --update --no-cache openssl1.1-compat` line needs also to be added to the runner. Without, I received an error stating that the connection to the database was not possible:

```
yarn run v1.22.19
$ npm-run-all check-db update-tracker start-server
$ node scripts/check-db.js
✓ DATABASE_URL is defined.
✗ Unable to connect to the database.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "check-db" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

> @d3ward @mikecao this is the actual issue and fix: [prisma/prisma#16553 (comment)](https://github.com/prisma/prisma/issues/16553#issuecomment-1353302617)
> 
> I added that `apk install` line to both the builder and the runner in the Dockerfile, though I'm not sure it's actually needed in the runner.
> 
> EDIT: looks like someone has opened PR for it as well: #1708

**Comment by gregonarash** [comment]
If someone runs into this issue - Prisma + Alpine version is  the problem. Likely just updating Prisma to 4.8.0 will fix it.

https://github.com/umami-software/umami/pull/1708
